### PR TITLE
Scripts/Spells: Migrate More Scripted Spells to Scripts

### PR DESCRIPTION
**Pull request by Sorikoff**
**Changes proposed:**
Migrate even more spells to scripts system. This PR brings very minimal changes, since it is not the aim of this PR.

**Target branch(es):** 3.3.5

**Tests performed:** Tested each migrated spell.

### CODE REVIEW

**Comment by jackpoz** [comment]
As I had a bad experience with https://github.com/TrinityCore/TrinityCore/pull/22332 that caused https://github.com/TrinityCore/TrinityCore/issues/21435 (and I had to do https://github.com/TrinityCore/TrinityCore/pull/23228 ) , could you check that not only the spells don't crash worldserver but that every little detail about them still works ?

File: src/server/scripts/Spells/spell_priest.cpp [hunk 1]
```diff
@@ -678,6 +679,43 @@ class spell_pri_item_t6_trinket : public SpellScriptLoader
...
 
+// 60123 - Lightwell
+class spell_pri_lightwell : public SpellScript
+{
+    PrepareSpellScript(spell_pri_lightwell);
+
+    void HandleScriptEffect(SpellEffIndex /* effIndex */)
+    {
+        Unit* caster = GetCaster();
+        if (!caster->ToCreature()->IsSummon())
```
Comment (jackpoz) [inline src/server/scripts/Spells/spell_priest.cpp]: please don't assume that caster is a Creature

**Comment by jackpoz** [comment]
Actually, thinking about it, this PR changes 8 spells which are 7 too many for a PR.
Please open a PR moving only 1 spell at a time so that it can be clearly reviewed ( see for example https://github.com/TrinityCore/TrinityCore/pull/23454#discussion_r295807680 , it's a pain to have to scroll up and time to compare which script is related to which spell and see if the code is the same)

**Comment by Sorikoff** [comment]
Yeah, this makes sense. Will make individual PRs.

**Pull request closed without merging**